excel-export-service/README.md

```diff
@@ -4,21 +4,7 @@ The excel export service is responsible for exporting multichain's data to a hum
 
 ## Environment Variables
 
```

Comment:
- [ ] Similarly as the email-service, the getting started section mentions non existing docker-compose - rather link the documentation in the `/docker-compose` folder
- [ ] remove `startDev.sh` script
- [ ] move `Environment Variables`section to `getting started` and also add more info like "in order to activate and configure the excel-export feature in TB you ened to set some env vars ..."